chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/test/InstanceSupervisor.kt

```diff
@@ -46,7 +43,8 @@ object InstanceSupervisor {
      * Tear downs the instance and runs the provided actions.
      */
     fun setUpInstanceWith(vararg actions: InstanceAction) {
-        waitForIdleInstance().wait(Wait.For(60, TimeUnit.SECONDS))
+        waitForIdleInstance()
+                // .wait(Wait.For(60, TimeUnit.SECONDS))
```

Comment:
what's the motivation to comment out this instruction?

Reply:
Hmm, I can't remember now 😅I presume it broke for me at some point. I've uncommented it so we have it back.